pkg/services/searchV2/http.go

```diff
@@ -57,7 +51,27 @@ func (s *searchHTTPService) doQuery(c *models.ReqContext) response.Response {
 		return response.Error(400, "error parsing body", err)
 	}
 
-	resp := s.search.doDashboardQuery(c.Req.Context(), c.SignedInUser, c.OrgID, *query)
+	var resp *backend.DataResponse
+	if true {
+		resp = s.fallback.doSQLQuery(c, query)
+	} else {
+		searchReadinessCheckResp := s.search.IsReady(c.Req.Context(), c.OrgID)
```

Comment:
Looking at this more... maybe the fallback should be inside the service directly?

That would let us skip the frontend hoops